ui/package.json

```
@@ -4,7 +4,7 @@
   "version": "0.0.0",
   "type": "module",
   "scripts": {
-    "dev": "export VITE_HOSTNAME=$CODESPACE_NAME && export VITE_GITHUB_CODESPACES_PORT_FORWARDING_DOMAIN=$GITHUB_CODESPACES_PORT_FORWARDING_DOMAIN && vite",
+    "dev": "bash -c ./scripts/dev.sh",
```

```suggestion
    "dev": "VITE_HOSTNAME=${CODESPACE_NAME:-} VITE_GITHUB_CODESPACES_PORT_FORWARDING_DOMAIN=${GITHUB_CODESPACES_PORT_FORWARDING_DOMAIN:-} vite",
```
I believe this solves the issue without the need of a script. WDYT?

---

Perfect!

---

@mujahidkay removed the script file too.